e2e/pnpm_lockfiles/MODULE.bazel

```diff
@@ -79,7 +81,7 @@ npm = use_extension(
         # Deep with many peers
         "lock-%s__at_rollup_plugin-typescript__8.2.1__%s" % (
             version,
-            "ommloj5qql5ba6x5wuiluawhoi" if version == "v54" else "1662662592",
+            "ommloj5qql5ba6x5wuiluawhoi" if version == "v54" else "1662662592" if version == "v90" else "1813138439",
```

Comment:
pnpm9 did not change the `@rollup/plugin-typescript` peer `rollup` version, pnpm <9 did resolve the peer differently 🤷